config/locales/server.en.yml

```
@@ -3842,6 +3842,14 @@ en:
 
         Enjoy your stay!
 
+    signup_after_reject:
+      title: "Signup After Reject"
+      subject_template: "You've been rejected on %{site_name}!"
```

```suggestion
      subject_template: "You've been rejected on %{site_name}"
```
I feel like the `!` might feel like we're rubbing it in or something. Better to be more neutral.